src/test/java/com/google/sps/servlets/AggregationServletTest.java

```diff
@@ -33,6 +37,13 @@
 @RunWith(JUnit4.class)
 public final class AggregationServletTest {
 
+  private final String LOGIN_URL = "/login";
```

Comment:
Please see a previous comment that, for such constants that applies to both implementation files and test files, it's better to define the constants in the implementation file (in this case AggregationServlet.java). And replace the hard-code strings there (e.g. "/login") by the constant, and refer the same constant here in the test file.

Reply:
I have made this change to the servlets, but also in the next PR, per Chuxiang's suggestion, I'm looking to additionally do a test util of some kind that will have very common names stored within.